### PR TITLE
Add support for the custom_info object as OrderRequest argument

### PR DESCRIPTION
**Pull request by DaanDeSmedt**
# Description

As per the documentation provided by MultiSafepay (https://docs.multisafepay.com/reference/createorder), the `custom_info` object allows the inclusion of extra order-related detail fields.

I've sourced the php-sdk but have not come across any indication of this feature.

To address this gap, I've taken the liberty in creating this PR to suggest some code changes.
Which addresses an approach for reported  [Support custom_info when creating a payment page/link #8](https://github.com/MultiSafepay/php-sdk/issues/8)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Added Unit tests

Added basis unit tests to check correct implementation.

- Added test `testRequestOrderRequestWithCustomInfo`on the test class `OrderRequestTest.php`
- Added Testcase `CustomInfoTest` to validate setting & retrieving custom info based on the `custom_1`, `custom_2` & `custom_3` approach described in the docs.

### CODE REVIEW

**Comment by Daniel-MultiSafepay** [comment]
Hi @DaanDeSmedt 

Thanks for submitting this contribution.  We just assigned two developers as reviewers of this PR. 
They will help and work to pass the tests before merging this PR.

**Comment by DaanDeSmedt** [comment]
@Daniel-MultiSafepay , 

Thanks for the heads-up. 👍 

I've fixed the Code sniffer / Check coding conventions on my added code. 

I suppose some extra pair of eyes on the failing unit tests would be welcome, since these failing unit tests don't seem related to my additions(?).

Have a great day!

File: src/Api/Transactions/OrderRequest/Arguments/CustomInfo.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+     */
+    public function getData(): array
+    {
+        return [
+            'custom_1' => $this->getCustom1(),
+            'custom_2' => $this->getCustom2(),
+            'custom_3' => $this->getCustom3(),
+        ];
+    }
+}
```
Comment (Mike-MultiSafepay) [inline src/Api/Transactions/OrderRequest/Arguments/CustomInfo.php:104]: Expected 1 newline at end of file; 0 found

**Review by Mike-MultiSafepay — CHANGES_REQUESTED** [review]
Hi @DaanDeSmedt 

After reviewing your code we can confirm that the changes are working as intended. There is only an issue with the code sniffer. If you could solve that issue we can merge the PR.

Regarding the tests failing, this is due to not having certain payment methods available on your test account. I've added an internal ticket to resolve this issue. 

So for the failing tests, you don't have to do anything at this moment

File: src/Api/Transactions/OrderRequest/Arguments/CustomInfo.php (hunk 1 above)
Comment (DaanDeSmedt) [reply src/Api/Transactions/OrderRequest/Arguments/CustomInfo.php]: @Mike-MultiSafepay , great feedback, thanks! Just pushed a fix for the code sniffer issue mentioned.

**Comment by DaanDeSmedt** [comment]
You're welcome, and thank you for merging the PR into the master branch. 

I appreciate your offer of a present as a token of appreciation for my contribution.
I'll send an email to [support@multisafepay.com](mailto:support@multisafepay.com), referencing this PR.

Great day!

**Pull request merged**